src/calculator/UtilsClass.java

```diff
@@ -61,6 +100,17 @@ public static void printResult(double answer) {
             Date date = new Date(System.currentTimeMillis());
 
 
+            //print to results column in my remotemysql db
+            Connection con = getDbConnection();
+            try {
+                Statement st = con.createStatement();
+                st.execute("INSERT INTO `calculator_db` (`" + RESULT_COLUMN + "`) VALUES ('" + answer + "')");
```

Comment:
Great job!
Please replace with prepared statements:
https://docs.oracle.com/javase/tutorial/jdbc/basics/prepared.html

Reply:
Done

Reply:
I don't see the prepared statement object.. did you forget to commit it?